openbas-front/src/utils/Charts.js

```diff
@@ -273,6 +275,8 @@ export const verticalBarsChartOptions = (
   isResult = false,
   isFakeData = false,
   max = undefined,
+  isChartAPlaceholder = false,
```

Comment:
Can we maybe use the existing isFakeData property instead of adding a new one ?

Reply:
If we use `isFakeData` directly we are going to also impact the `Simulations` chart in the home dashboard.
The scope described in the issue is only to impact the `Simulations results` chart at the scenario dashboard level.

Should we also modify the `Simulations` chart ? cc @jborozco

Reply:
Discussed with @jborozco, we are updating also the `Simulations` chart